pkgs/development/compilers/arocc/package.nix

```diff
@@ -13,13 +13,23 @@ stdenv.mkDerivation (finalAttrs: {
 
   nativeBuildInputs = [ zig.hook ];
 
+  outputs = [
+    "out"
+    "lib"
+  ];
```

Comment:
This needs more explanation (in a comment or the commit message).

Reply:
Not sure how to explain more than just "it's needed for eval"

Reply:
Why is it needed for eval?

Reply:
I forget where exactly, I think it's either with stdenv or the CC wrapper which is expecting a lib output in the CC.

Reply:
Oh it turns out that it's just a lot of packages expect `stdenv.cc.cc.lib` to exist. Dwarf Fortress is one such notable example which would fail eval without it. `pkgsArocc.ArchiSteamFarm` is one which would fail eval early.

Reply:
I've updated the commit description and comment.

Reply:
Great, thanks. I'm not sure this is a good idea though — some of the packages are looking for specific paths under the lib output, and those will probably be silently broken at runtime if we do it this way.  I think it's probably worth the few seconds per file × ~200 files it would take to conditionalize packages that just take it as build inputs, to not have silently broken packages.  There's also not really any need to get everything to eval right now — if you'd rather just fix the packages you care about, and leave the rest for later, that'd be okay too — better to have things be broken loudly than quietly.

Reply:
Hmm, so just remove the fake output and we can fix those packages in a future PR?

Reply:
That would be my preference.

Reply:
Cool, dropped that change.